### PR TITLE
janitorial: drop repeated words words

### PR DESCRIPTION
**Pull request by dankenigsberg**
Done via
```
sed -i 's/\b\([a-z]\+\) \1\b/\1/' \
   $(git grep -l '\b\([a-z]\+\) \1\b' build config image kubecontrolplane legacyconfig)
```

### CODE REVIEW

**Comment by bparees** [comment]
@dankenigsberg thanks for the cleanup, looks like you may have missed a change in the types.go for the monitoring update which is causing the verify job to fail, fix that up and i think this is good to go.

/approve

File: image/v1/types.go [hunk 1]
```diff
@@ -651,7 +651,7 @@ type RepositoryImportStatus struct {
 type ImageImportSpec struct {
 	// From is the source of an image to import; only kind DockerImage is allowed
 	From corev1.ObjectReference `json:"from" protobuf:"bytes,1,opt,name=from"`
-	// To is a tag in the current image stream to assign the imported image to, if name is not specified the default tag from from.name will be used
+	// To is a tag in the current image stream to assign the imported image to, if name is not specified the default tag from.name will be used
```
Comment (dmage) [inline image/v1/types.go:654]: > from _from.name_

It doesn't look like a typo.

**Comment by dankenigsberg** [comment]
> @dankenigsberg thanks for the cleanup, looks like you may have missed a change in the types.go for the monitoring update which is causing the verify job to fail, fix that up and i think this is good to go.

It seems that it would take me some time to get `make verify` to pass. I do not comprehend which of the many errors reported in the [log](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_api/1070/pull-ci-openshift-api-master-verify/1464127963267076096) is at fault.

**Pull request merged**